model/result.ts

```diff
@@ -1,10 +1,11 @@
-import { z } from "zod";
+import { date, z } from "zod";
 
-export enum Result {
-  WIN = "You Win!!!",
-  LOSE = "You lose :< ",
-  DRAW = "It's a Draw!",
-}
+export const results = z.object({
```

Comment:
```suggestion
export const result = z.object({
```
se è uno solo

Inoltre tendiamo come convenzione a usare l'iniziale maiuscola per gli schemi zod così chiamiamo uguali schema e tipo. In questo modo si possono importare in un colpo solo. Quindi si potrebbe chiamare questo `Result` e sotto `ResultArray`. Anche se non è una convezione che seguiamo sempre.